app/auth/forms.py

```diff
@@ -78,4 +78,10 @@ class ResetPasswordRequestForm(FlaskForm):
 class ResetPasswordForm(FlaskForm):
     password = PasswordField('Password', validators=[DataRequired()])
     password2 = PasswordField('Repeat Password', validators=[DataRequired(), EqualTo('password')])
-    submit = SubmitField('Password Reset')
\ No newline at end of file
+    submit = SubmitField('Password Reset')
+
+class ResetPasswordFormPreferences(FlaskForm):
```

Comment:
I think the FlaskForm convention is to have the last word be Form in class name. 

You could probably just change this to ChangPasswordForm(). So it's different than ResetPasswordForm().